user/mongoStoreClient_test.go

```diff
@@ -176,13 +170,13 @@ func TestMongoStoreTokenOperations(t *testing.T) {
 	mgoTokensCollection(cpy).DropCollection()
 
 	if err := mgoTokensCollection(cpy).Create(&mgo.CollectionInfo{}); err != nil {
-		t.Fatalf("We couldn't created the token collection for these tests ", err)
+		t.Fatal("We couldn't created the token collection for these tests ", err)
 	}
 
 	/*
 	 * THE TESTS
 	 */
-	sessionToken, _ := CreateSessionToken(TD, FAKE_SECRET)
+	sessionToken, _ := CreateSessionToken(testing_token_data, TokenConfig{DurationHours: 12, Secret: testing_fake_secret})
```

Comment:
Do you want to make the 12 a variable as well, like testing_duration_hours?  It is okay to leave as-is, though.